packages/docs/src/routes/api/qwik/index.md

```diff
@@ -3679,7 +3679,7 @@ Options for the prefetch service worker.
 </tbody></table>
 **Returns:**
 
-[JSXNode](#jsxnode)&lt;'script'&gt;
+JSXNode&lt;'script'&gt;
```

Comment:
Can you run `pnpm api.update` so it doesn't do this? Something in the build causes this, I wish it didn't :(